### ISSUE TITLE
search is deprecated

### ISSUE THREAD

**Issue by pkitatta** [ionic-team/ionic-starter-super#134]
![image](https://user-images.githubusercontent.com/17586059/30958136-e444b2b6-a444-11e7-850e-d84747a4f887.png)

**Comment by janpio** [comment]
What exactly?
Why is `search` stikethrough in your IDE?

**Comment by pkitatta** [comment]
It says it is deprecated.

**Comment by janpio** [comment]
What exactly? The `search` property of `options`?

Here is the code you screenshotted:
https://github.com/ionic-team/ionic-starter-super/blob/b75cf773e430da0f1d4837926b064448bc9f88cb/src/providers/api/api.ts#L29

So `options` are  `RequestOptions` from `@angular/http`. Can you go into there and see what exactly it says?

In general: Deprecated only means that it will be removed in the future. For you, using the code right now, this is no problem at all.

**Comment by pkitatta** [comment]
Yes the `search` property of options

`Deprecated symbol used, consult docs for better alternative less... (Ctrl+F1) 
Checks for using deprecated JavaScript functions and variables . The validation works in JavaScript, html, or jsp files`

**Comment by janpio** [comment]
So what do the `@angular/http` docs or code say about alternatives?

**Comment by pkitatta** [comment]
I doesn't When I click on the search word, that's what I get. I am using webstorm and I am not so advanced. Maybe it's just an IDE thing.

**Comment by janpio** [comment]
Which version of `@angular/http` are you using? Should be in `packages.json`.

**Comment by pkitatta** [comment]
4.1.3

**Comment by pkitatta** [comment]
BTW good starter app. It's bringing me up to speed with ionic-angular.

I request your help with my endpoints.

My api endpoint is `http://watsapp.dev`
so I added it to 
`url: string = 'https://watsapp.dev/events';`
but am not sure how I should deal with this:
`return this.http.get(this.url + '/' + endpoint, options);`
with respect to my endpoint which is:
`return this.http.get(this.url + '/' + endpoint, options);`

Any help?

**Comment by janpio** [comment]
The API calls get the endpoint as a param, like here:
https://github.com/ionic-team/ionic-starter-super/blob/b75cf773e430da0f1d4837926b064448bc9f88cb/src/providers/items/items.ts#L16

**Comment by pkitatta** [comment]
I am at the api.ts file itself and trying to connect to my laravel api end point.

I added `event` to `get(event: string, params?: any, options?: RequestOptions) {`. Is this correct?

**Comment by janpio** [comment]
No. Please ask these really basic questions at the Ionic community forum: https://forum.ionicframework.com/

**Comment by pkitatta** [comment]
Aright, thanks for help.
